syntax = "proto3";

package PFDESolverServer;

import "config.proto";
import "result.proto";

service TFDESolverServer {
    rpc RunTask(PFDESolver.TClientConfig) returns (PFDESolver.TResult) {}
}
